docs/products/opensearch/dashboards/getting-started.rst

```diff
@@ -0,0 +1,119 @@
+Getting started
+===============
+
+To start using Aiven for OpenSearch Dashboards, you should :doc:`create Aiven for OpenSearch service first</docs/products/opensearch/getting-started>` and OpenSearch Dashboards service will be created alongside it. Once the Aiven for OpenSearch service is running you can find connection information to your OpenSearch Dashboards in the service overview page.
+
+.. note::
+    If you are using Chrome, you need to enter **user name** and **password** manually. :doc:`Read more <howto/access-from-chrome>` for details.
```

Comment:
Why are we mentioning this? What happens with other browsers?

Reply:
True, I'm checking now FF and it is same